### PR TITLE
Add CI for Postgresql 12 / Postgis 3.2 on Ubuntu 20.04

### PR DESCRIPTION
**Pull request by pblottiere**

### CODE REVIEW

**Comment by pblottiere** [comment]
@lbartoletti opinions?

File: .github/workflows/focal_postgres12_postgis3.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+jobs:
+  build:
+    name: Building and testing
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Install PostgreSQL and PostGIS
+        env:
+          POSTGRESQL_VERSION: 12
```
Comment (lbartoletti) [inline .github/workflows/focal_postgres12_postgis3.yml:18]: And PG14?

File: .github/workflows/focal_postgres12_postgis3.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          POSTGIS_VERSION: 3
+        run: .github/scripts/postgresql_postgis.sh
+      - name: Install and check PgPointCloud
+        env:
+          POSTGRESQL_VERSION: 12
+          POSTGIS_VERSION: 3
```
Comment (lbartoletti) [inline .github/workflows/focal_postgres12_postgis3.yml:24]: 3 is for 3.0, 3.1 or 3.2?

File: .github/workflows/focal_postgres12_postgis3.yml (hunk 1 above)
Comment (pblottiere) [reply .github/workflows/focal_postgres12_postgis3.yml:18]: It'll be in another PR

File: .github/workflows/focal_postgres12_postgis3.yml (hunk 2 above)
Comment (pblottiere) [reply .github/workflows/focal_postgres12_postgis3.yml:24]: With these environment variables the next package is installed: `postgresql-$POSTGRESQL_VERSION-postgis-$POSTGIS_VERSION`. So in this case it's 3.2.

Comment (pblottiere) [reply .github/workflows/focal_postgres12_postgis3.yml:24]: (i'm going o update the tag and the doc accordingly)

**Review by lbartoletti — APPROVED** [review]

**Pull request merged**